### ISSUE TITLE
xss fuzzMouse fails because of functions in BASEA11yStrings.js

### ISSUE THREAD

**Issue by zepumph** [phetsims/balloons-and-static-electricity#328]
@jessegreenberg bayes is failing pretty regularly because xss tests assume that all keys in the a11y strings files are strings. 

My two thoughts:
1. If these functions are just for testing, then maybe make an ignore list, I see you have done this with one but not the other:
```js
  if ( phet.chipper.queryParameters.stringTest === 'xss' ) {
    for ( var key in BASEA11yStrings ) {
      if ( BASEA11yStrings[ key ] !== BASEA11yStrings.stripPlaceholders ) {
        BASEA11yStrings[ key ] += '/* blarg */ ';
      }
    }
  
```
2. Promote these to functions in StringUtils, maybe with an a11y prefix or a11y doc?

Let me know if you want help.  

![image](https://user-images.githubusercontent.com/6856943/32925114-b39f2e92-caec-11e7-904d-5318903b0b4f.png)

**Comment by jessegreenberg** [comment]
@zepumph thanks for catching this! I went with a whitelist for now in the above commit, unable to reproduce the bug.

> Promote these to functions in StringUtils, maybe with an a11y prefix or a11y doc?

This is a great idea, this should definitely be done if these functions are going to stick around. They have been very helpful for managing really long string patterns and punctuation with several placeholders.

But they could be bad news for i18n, and maybe a different pattern would be better. Closing this issue since BASE should be good on bayes, ill creat another issue to talk about `stripPlaceholders`.
